packages/react/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -25,7 +25,7 @@ export interface DropdownSelectedItemProps extends UIComponentProps<DropdownSele
   active?: boolean
 
   /** Header of the selected item. */
-  header?: string
+  header?: ShorthandValue
```

Comment:
Be careful with this one, as the Label's content is not shorthand value. You may need to create the element here before sending it to the `Label` component.

Reply:
thank you, will take a look.I just drafted changes for now, as People Team requested a change here.